src/styles/blocks/screen.scss

```
@@ -0,0 +1,12 @@
+.stop-watch {
+  position: relative;
+  box-sizing: border-box;
+  width: $clock;
+  height: $clock;
+  border: ($clock / 2 - $arrow-height - 2) solid #8d0d4d;
+  background-color: #000;
+  border-radius: 50%;
+  top: 50%;
+  left: 50%;
+  margin-top: 150px;
```

group it
```suggestion
  position: relative;
  top: 50%;
  left: 50%;
  
  box-sizing: border-box;
  width: $clock;
  height: $clock;
  margin-top: 150px;
  
  border: ($clock / 2 - $arrow-height - 2) solid #8d0d4d;
  background-color: #000;
  border-radius: 50%;
```
see [recommendations](http://codeguide.academy/html-css.html#css-order)